src/Dropdown/Dropdown.test.jsx

```diff
@@ -14,20 +14,16 @@ const props = {
 };
 
 const menuOpen = (isOpen, wrapper) => {
-  expect(wrapper.hasClass('show')).toEqual(isOpen);
-  expect(wrapper.find('[type="button"]').prop('aria-expanded')).toEqual(isOpen);
+  expect(wrapper.childAt(0).hasClass('show')).toEqual(isOpen);
```

Comment:
Interesting -- so a shallow render no longer returns a direct reference to the DOM node?